shelf-layouts/Shortcuts and adlib scroll.json

```
@@ -14,7 +14,7 @@
       "default": false,
       "x": 1,
       "y": 7,
-      "width": 64.5,
+      "width": 65.5,
```

These should be relative to the right-aligned static buttons panel which has a fixed width. To make these panels fit not just at a fixed resolution, but scale accordingly, you can set the width to `-15.7 - 1 (for the static buttons' x) - 1 (for a margin) = -17.7` and you'll have SOF-1084 out of the way
```suggestion
      "width": -17.7,
```

---

That explains why I have been seeing negative numbers in layouts.
I have been quite confused about that since it's not intuitive that a negative number would work that way.
I'll give it a try in gallery D before updating the PR :)

---

Updated